nav2_msgs/action/BackUp.action

```diff
@@ -3,6 +3,7 @@
 geometry_msgs/Point target
 float32 speed
 builtin_interfaces/Duration time_allowance
+bool no_collision_checks
```

Comment:
For both: set default to `False`

Can we also rename it here + elsewhere as `disable_collision_checks`?